Sources/Datadog/FeaturesIntegration/WebView/WebLogEventConsumer.swift

```diff
@@ -7,7 +7,70 @@
 import Foundation
 
 internal class WebLogEventConsumer: WebEventConsumer {
-    func consume(event: [String: Any], eventType: String) {
-        // TODO: RUMM-1791 implement event consumers
+    private struct Constants {
+        static let logEventType = "log"
+        static let internalLogEventType = "internal_log"
+
+        static let applicationIDKey = "application_id"
+        static let sessionIDKey = "session_id"
+        static let ddTagsKey = "ddtags"
+        static let dateKey = "date"
+    }
+
+    private let userLogsWriter: Writer
+    private let internalLogsWriter: Writer?
+    private let dateCorrector: DateCorrectorType
+    private let rumContextProvider: RUMContextProvider?
+    private let jsonDecoder = JSONDecoder()
+
+    private lazy var ddTags: String = {
+        let versionKey = LogEventEncoder.StaticCodingKeys.applicationVersion.rawValue
+        let versionValue = LoggingFeature.instance?.configuration.common.applicationVersion ?? ""
+        let envKey = LogEventEncoder.StaticCodingKeys.environment.rawValue
+        let envValue = LoggingFeature.instance?.configuration.common.environment ?? ""
```

Comment:
No reason to consider `envValue` and `versionValue` if `LoggingFeature.instance` is `nil` (if the feature is disabled). Can't we use dependency injection to make the solution depend explicitly on `LoggingFeature` instance? It will be simpler (less control flows), less error prone (no optionals) and will result with better tests coverage.